mewarpx/mewarpx/diags_store/diag_base.py

```diff
@@ -9,8 +9,7 @@
 
 from mewarpx.mwxrun import mwxrun
 
-# Get module-level logger
-logger = logging.getLogger(__name__)
+from mewarpx.logger import logger
```

Comment:
Please keep the initial code here. That ensures each python module has its logs printed out with its module name.